cmd/antrea-agent/agent.go

```diff
@@ -133,11 +133,12 @@ func run(o *Options) error {
 		encryptionMode = config.TrafficEncryptionModeIPSec
 	}
 	networkConfig := &config.NetworkConfig{
-		TunnelType:            ovsconfig.TunnelType(o.config.TunnelType),
-		TrafficEncapMode:      encapMode,
-		TrafficEncryptionMode: encryptionMode,
-		TransportIface:        o.config.TransportInterface,
-		TransportIfaceCIDRs:   o.config.TransportInterfaceCIDRs,
+		TunnelType:                  ovsconfig.TunnelType(o.config.TunnelType),
+		TrafficEncapMode:            encapMode,
+		TrafficEncryptionMode:       encryptionMode,
+		TransportIface:              o.config.TransportInterface,
+		TransportIfaceCIDRs:         o.config.TransportInterfaceCIDRs,
+		ExternalMulticastInterfaces: o.config.ExternalMulticastInterfaces,
```

Comment:
Should ExternalMulticastInterfaces defaults to the primary network interface (the transport interface) if not set? otherwise user would have to set it even if there is single interface, which might be the most common case.